app/Http/Controllers/User/MyRequestController.php

```diff
@@ -81,6 +84,20 @@ public function update(Request $request, $id)
             $this->notification->store($data);
         }
 
+        if ($request->status == 'returning') {
+            $book_id = $this->bookUser->findId($id)->book_id;
+            $count_returnedBook = $this->bookUser->countReturned($book_id);
+            $find_Returned = $this->bookMeta->findReturned($book_id);
+            if (empty($find_Returned)) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/User/MyRequestController.php</br>**Line**: 91</br>Variable "find_Returned" is not in valid camel caps format